news/2022-09-01-osumonthly-2.md

```diff
@@ -151,7 +151,7 @@ Prepare for trouble! Make it double! Be the first one to correctly answer both o
 - If you are a fan of doujin music, check out the [Taishi Appreciation Team Beatmapping Contest (TATBC) 2022](https://osu.ppy.sh/community/forums/topics/1631506?n=1) hosted by [-Tochi](https://osu.ppy.sh/users/3664366) and [Lince Cosmico](https://osu.ppy.sh/users/6070370). The songs to be used for the mapping competition were only recently announced, so you'll have all the time you need to get into the mapping groove.
 - Submissions for the [osu! Featured Artist Mapping Contest (FAMC) 2022](https://osu.ppy.sh/community/forums/topics/1619393?n=1) hosted by [Mafumafu](https://osu.ppy.sh/users/3076909) and [pishifat](https://osu.ppy.sh/users/3178418) are open until the 25th of September, so as they say, better late than never!
 - The [15th Anniversary Art Contest](https://osu.ppy.sh/home/news/2022-08-15-15-anniversary-art-contest) has just closed its submissions, and we should consequently see the votes coming in very soon. So if you have even just five minutes, don't forget to cast your votes!
-- osu!'s most recent skinning competition with the theme of [*Chromatic Alteration*](https://osu.ppy.sh/home/news/2022-07-18-skinning-contest-chromatic-alteration-announcement) has just entered it's voting phase — take a look at all the wonderful creations and support them with your votes!
+- osu!'s most recent skinning competition with the theme of [Chromatic Alteration](https://osu.ppy.sh/home/news/2022-07-18-skinning-contest-chromatic-alteration-announcement) has just entered it's voting phase — take a look at all the wonderful creations and support them with your votes!
```

Comment:
can be in italics imo

Reply:
it was unintended unfortunately, also kinda triggers my OCD 
![image](https://user-images.githubusercontent.com/24534022/187946534-a430a5f5-4910-469e-98e4-ece734e1e574.png)